external-crates/move/crates/move-compiler/src/command_line/compiler.rs

```diff
@@ -347,7 +339,7 @@ impl Compiler {
 
         source_text
             .iter_mut()
-            .for_each(|(_, (path, _))| *path = relativize_path(&vfs_root, *path));
+            .for_each(|(_, (path, _))| *path = vfs_to_original_path[path]);
```

Comment:
This map access should not be unconditional as new paths can be generated between when the map is initialized and when it's used here, so not every path will have an entry in the map.

Reply:
yeah I found out as much from tests. Attempting to fix but it might blow up on windows